beesdoo_worker_status/tests/test_beesdoo_shift.py

```diff
@@ -42,6 +42,9 @@ def setUp(self):
     def test_shift_counters(self):
         "Test shift counters calculation and cooperative status update"
 
+        config = self.env["ir.config_parameter"].sudo()
+        config.set_param("irregular_penalty", True)
+
```

Comment:
Let's put this in the setUp function and get rid of it here and elsewhere. We simply set it to False for exceptional cases.

Reply:
done